internal/installutil/helm/helm.go

```diff
@@ -0,0 +1,195 @@
+package helm
+
+import (
+	"fmt"
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/waypoint/builtin/k8s"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/release"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/client-go/discovery"
+	memcached "k8s.io/client-go/discovery/cached/memory"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+	"k8s.io/client-go/tools/clientcmd"
+	"net/url"
+	"strings"
+)
+
+const DefaultHelmChartVersion = "0.1.8"
```

Comment:
This will need to be updated each time we release a new version of Waypoint and update the version of our Helm chart. Does Helm not have an understanding of `latest`? I'm going to put some code suggestions over in `internal/runnerinstall/k8s.go`.

Reply:
That is a super good idea. Especially because if we mess up releasing a version of Helm, we'd now have to do yet-another release of Waypoint too.

Reply:
Wow, the helm Golang package does not make this easy, do they.